### DISCUSSION TITLE
Online Players bot function for discord

### DISCUSSION THREAD

**Discussion by elusive-ttv** [gameserverapp/Platform#186] [Suggestions]
Since we are able to use the !online command to see online players, it would be really cool to have something like the server status cards that we can put in an 'online right now' page or something for the playerbase to see who and how many people are online.

**Comment by mvaessen** [comment]
~~This is now available~~ nvm, i misread your request

**Comment by mvaessen** [comment]
Thanks for your suggestion! This is now available, as `Discord dynamic cards`.
